packages/app/src/vis-packs/core/matrix/config.tsx

```diff
@@ -24,10 +28,15 @@ function createStore() {
         setCustomCellWidth: (customCellWidth) => {
           set({ customCellWidth });
         },
+
+        notation: Notation.Auto,
```

Comment:
Should we keep scientific as the default? This is our first user feedback on this matter, so not sure if other users wouldn't prefer keeping the current behaviour. 🤷

Reply:
Good call :+1: